### ISSUE TITLE
Audio File Wizard ArgumentOutofRangeException when the Output Folder field is valueless

### ISSUE THREAD

**Issue by blawnode** [jackyyang09/Simple-Unity-Audio-Manager#42]
### Bug Description
Argument Out of Range exception in JSAMEditorHelper.GenerateFolderStructureAt().
Occurs in the Audio File Wizard window, when attempting to generate audio file objects, with no value in the Output Folder field.
A cryptic exception occurs:

```
ArgumentOutOfRangeException: Index and count must refer to a location within the string.

Parameter name: count

System.String.Remove(...)

JSAM.JSAMEditor.JSAMEditorHelper.GenerateFolderStructureAt(... /JSAMEditorHelper.cs:196)
```

A much more suitable approach that is not cryptic, may be a message within the Wizard window, below the button,
Or a simple Console print can do too,
Which will tell the user that the output folder must be given a value.

### Steps to Reproduce
Open Audio File Wizard window
Make sure the Output Folder field has no value.
Press Generate Audio File Objects.
Cryptic exception occurs.
(May need affirmation.)

### Unity Versions Tested With:
2021.3

**Comment by jackyyang09** [comment]
I wasn't able to reproduce the issue on my side. What version of JSAM are you using? I don't think this issue should be occurring in the latest version.

**Comment by blawnode** [comment]
JSAM v3.0.0-preview.6. Unity 2021.3.33f1 in particular.
Just reproduced it on the same project I've had this issue with.
When this happens, a `Hold On... (busy for ?s)... Importing assets` window also appears.

(Note that the first error is about com.unity.toolchain.win-x86_64-linux-x86_64, and doesn't cause further issues. It's probably irrelevant to this JSAM issue.)

![image](https://github.com/user-attachments/assets/3163d899-c6f1-40d6-a831-d35ac54771a1)

**Comment by jackyyang09** [comment]
This issue was fixed as of 3.0.1. The latest release version is 3.0.2.

I'd recommend updating your local package

**Comment by blawnode** [comment]
Ah, fantastic! I haven't seen this in the issue list so it's nice to know it's resolved :p
Tested and you're right, the button becomes disabled until an Output Folder is chosen.
Thank you!
